satpy/readers/__init__.py

```diff
@@ -538,3 +539,61 @@ def _get_reader_kwargs(reader, reader_kwargs):
         reader_kwargs_without_filter[k].pop('filter_parameters', None)
 
     return (reader_kwargs, reader_kwargs_without_filter)
+
+
+@total_ordering
+class FSFile(os.PathLike):
+    """Implementation of a PathLike file object, that can be opened.
+
+    This is made to be use in conjuction with fsspec or s3fs. For example::
+
+        zipfile = S3B_OL_2_WFR____20201103T100807_20201103T101107_20201103T121330_0179_045_179_1980_MAR_O_NR_002.zip
+        filename = "sentinel-s3-ol2wfr-zips/2020/11/03/" + filename
+        the_files = fsspec.open_files("simplecache::zip://**/*.nc::s3://" + filename, s3={'anon': False})
```

Comment:
You're passing `'anon': False` — might you have an example that is available with anonymous access such that any user can easily try out the example?

Reply:
I don't have open an example for olci files unfortunately, and this is the only reader supporting this right now.

Reply:
Fixed through switching to an abi example.